src/api/procedures/allowIdentityToCreatePortfolios.ts

```diff
@@ -0,0 +1,66 @@
+import { PolymeshError, Procedure } from '~/internal';
+import { ErrorCode } from '~/types';
+import { ExtrinsicParams, TransactionSpec } from '~/types/internal';
+import { stringToIdentityId } from '~/utils/conversion';
+
+/**
+ * @hidden
+ */
+export type AllowIdentityToCreatePortfoliosParams = {
+  did: string;
```

Comment:
I think its nice to allow passing in the entity directly. Lots of our APIs support this pattern already.

```suggestion
  identity: Identity | string
```

You can use the helper `asIdentity` to cast the union to an Identity object.